recipes-bsp/u-boot/files/0001-riscv32-Use-double-float-ABI-for-rv32.patch

```diff
@@ -0,0 +1,41 @@
+From 66dfe0fa886f6289add06d1af8642ce2b5302852 Mon Sep 17 00:00:00 2001
+From: Khem Raj <raj.khem@gmail.com>
+Date: Tue, 9 Feb 2021 16:40:12 -0800
+Subject: [PATCH] riscv32: Use double-float ABI for rv32
+
+So it can use libgcc built with OE toolchain
+Fixes
+error: "can't link hard-float modules with soft-float modules"
+
+Signed-off-by: Khem Raj <raj.khem@gmail.com>
+---
```

Comment:
Awesome!

Can this go upstream instead of here? Once the Linux-yocto build is working again I wanted to upstream RV32 support.